core/audits/server-response-time.js

```diff
@@ -61,6 +68,14 @@ class ServerResponseTime extends Audit {
     const mainResource = await MainResource.request({devtoolsLog, URL: artifacts.URL}, context);
 
     const responseTime = ServerResponseTime.calculateResponseTime(mainResource);
+    if (responseTime === null) {
```

Comment:
Add a test for this case?

Reply:
I'm not aware of a way to force a undefined timing with our unit testing.